PF-build.sh

```diff
@@ -329,7 +525,7 @@ do
 	fi
 	#Prepare hex files folders
 	if [ ! -d "$SCRIPT_PATH/../Hex-files/FW$FW-Build$BUILD/$MOTHERBOARD" ]; then
-		mkdir -p $SCRIPT_PATH/../Hex-files/FW$FW-Build$BUILD/$MOTHERBOARD || exit 10
+		mkdir -p $SCRIPT_PATH/../Hex-files/FW$FW-Build$BUILD/$MOTHERBOARD || exit 27
```

Comment:
Can the "Hex-files" folder be renamed to "PF-build-hex" or something similar to indicate it's for Prusa Firmware build?

Reply:
Sure it can be done.